jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -241,21 +241,19 @@ Test results, including timing data, will be saved in this subdirectory of the e
 - run:
       name: Test application
       command: |
-          TEST=$(circleci tests glob "src/__tests__/*.js" | circleci tests split --split-by=timings)
-          yarn test $TEST
+          TEST=$(circleci tests glob "src/__tests__/*.js")
+          echo "$TEST" | circleci tests run --command="xargs yarn test" --split-by=timings
 ----
 +
 This step uses the CircleCI CLI to pass in the location of the test suites and configure how the tests are split. You can use the `circleci tests glob` command to select the test files:
 +
 * First, you want those that match the `+src/__tests__/*.js+` globbing pattern, that is, any `.js` files located in `+src/__tests__+` and any of its subdirectories.
-* Then, the matching files are piped into `circleci tests split`, which creates the test split groupings.
-* The `--split-by=timings` flag indicates that the tests should be split according to timing data. For other test splitting options, consult the xref:parallelism-faster-jobs#splitting-test-files[Splitting test files section] of the Running rests in parallel page.
-+
-NOTE: The `circleci tests` commands (`glob` and `split`) cannot be run locally via the CLI as they require information that only exists within a CircleCI container.
+* Then, the matching files are piped into `circleci tests run`, which creates the test split groupings and then runs your tests.
+* The `--split-by=timings` flag indicates that the tests should be split according to timing data. For other test splitting options, see the xref:parallelism-faster-jobs#[Test splitting and parallelism] page.
```

Comment:
does the parallelism page enumerate all of the split-by options? and timings-type options?

Reply:
It does in the table of options for the tests run command but I think we could do with some content adding outside this to make it clearer, I'll do that